content/install-guides/windows-perf-vs-extension.md

```diff
@@ -0,0 +1,91 @@
+---
+### Title the install tools article with the name of the tool to be installed
+### Include vendor name where appropriate
+title: WindowsPerf Ecosystem - the Visual Studio Extension
+minutes_to_complete: 5
+
+official_docs: https://github.com/nader-zouaoui/windowsperf-vs-extension
+
+author_primary: Nader Zouaoui
+
+### Optional additional search terms (one per line) to assist in finding the article
+additional_search_terms:
+  - perf
+  - profiling
+  - profiler
+  - windows
+  - woa
+  - windows on arm
+  - open source windows on arm
+  - visual studio
+### FIXED, DO NOT MODIFY
+weight: 1 # Defines page ordering. Must be 1 for first (or only) page.
+tool_install: true # Set to true to be listed in main selection page, else false
+multi_install: FALSE # Set to true if first page of multi-page article, else false
+multitool_install_part: false # Set to true if a sub-page of a multi-page article, else false
+layout: installtoolsall # DO NOT MODIFY. Always true for tool install articles
+---
+
+## Introduction
+
+**WindowsPerf** is a lightweight performance profiling tool inspired by Linux perf, specifically tailored for Windows on ARM. It leverages the ARM64 PMU (Performance Monitor Unit) and its hardware counters to offer precise profiling capabilities. Recognizing the complexities of command-line interaction, the WindowsPerf GUI project is a Visual Studio 2022 extension that was initiated to provide a more intuitive, integrated within the development environment (IDE) experience. This tool enables developers to interact with WindowsPerf, adjust settings, and visualize performance data seamlessly within the IDE.
+
+## A Glimpse of the UI
+
+The WindowsPerf GUI extension is composed of several key features, each designed to streamline the user experience:
+
+![Sampling preview #center](../_images/wperf-vs-extension-sampling-preview.png)
+
+- **WindowsPerf Configuration**: Connect directly to `wperf.exe` for a seamless integration. Configuration is accessible via `Tools -> Options -> Windows Perf -> WindowsPerf Path`.
```

Comment:
I think you want to add here header like 

``## Installation`

This section is just cramped below the UI Glimpse picture.

![image](https://github.com/user-attachments/assets/37db2b48-ac0b-4652-8131-b14c659a8e60)

Reply:
Hmmm, this is hard to understand. So this is the list of things it does?
Not sure if this is right way to do it, it's a bulletpoint list of things, I thought I should follow these.
Maybe it's better to just write a paragraph explaining these things?